media/videostitcher/create_slate.go

```diff
@@ -45,12 +45,16 @@ func createSlate(w io.Writer, projectID, slateID, slateURI string) error {
 		},
 	}
 	// Creates the slate.
-	response, err := client.CreateSlate(ctx, req)
+	op, err := client.CreateSlate(ctx, req)
 	if err != nil {
 		return fmt.Errorf("client.CreateSlate: %w", err)
 	}
+	response, err := op.Wait(ctx)
+	if err != nil {
+		return fmt.Errorf("Wait: %w", err)
```

Comment:
nit: You can also just return the error without formatting. The string prefix provided doesn't add a lot of information.

Reply:
Done